checklist.md

```diff
@@ -10,9 +10,9 @@
 - [ ] Linting tools installed
 - [ ] RSpec installed
 - [ ] Suggested gems included
-- [ ] Sentry configured
+- [ ] Sentry, Appsignal, NewRelic configured depending on your choice.
+- [ ] Logs on Appsignal configured
```

Comment:
Appsignal configured twice?
Why not extract Appsignal from checklist.md:13 because we require logging to Appsignal?

Reply:
Yes, logging must be done on Appsignal. Monitoring can be done on other tools as well. @schmijos do you agree?

Reply:
Agreed. AppSignal is the default for logging.